src/NuGet.Core/NuGet.Protocol/EnhancedHttpRetryHelpers.cs

```diff
@@ -2,103 +2,110 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using NuGet.Common;
 
 namespace NuGet.Protocol
 {
     internal class EnhancedHttpRetryHelper
     {
-        private readonly IEnvironmentVariableReader _environmentVariableReader;
-        internal const string ExperimentalRetryEnabledEnvVarName = "NUGET_ENABLE_EXPERIMENTAL_HTTP_RETRY";
```

Comment:
Did they need to move to public?

I'm mostly asking not in the sense of, let's not, rather wouldn't that have made the footprint of the change smaller.

Reply:
If a class is `internal`, I lean towards making members like this `public`.  I don't feel strongly either way so its your call.

Reply:
I'm not too invested into it (this class won't change much likely). 

Just brought it up as someone that *could* make the total changeset smaller.